base/applications/cmdutils/taskkill/taskkill.h

```diff
@@ -35,3 +35,6 @@
 #define STRING_ENUM_FAILED      112
 #define STRING_TERMINATE_FAILED 113
 #define STRING_SELF_TERMINATION 114
+#define STRING_PARAM_TOO_MUCH   115
+#define STRING_INVALID_SYNTAX   116
+
```

Comment:
Why?

Reply:
> Why?
possibly my code editor made this...  I didn't mean to it. I will fix it later